frontend_vue/src/components/ui/IncidentDetails.vue

```diff
@@ -116,6 +116,10 @@ onMounted(() => {
   );
 });
 
+const formatIncidentDetails = (value: any) => {
```

Comment:
There's a file called incidentAlerService.ts we use to replace labels for the incident details. 

INCIDENT_DETAIL_CUSTOM_LABELS holds the keys we can replace, you can add 'loc' and IP there

Reply:
Thank you @vittoriaThinkst. This works much better!!